utils/ssh.py

```diff
@@ -262,15 +302,16 @@ def cpu_spike(self, seconds=60, cpus=2, **kwargs):
             "for ((i=0; i<instances; i++)) do while (($(date +%s) < $endtime)); "
             "do :; done & done".format(seconds, cpus), **kwargs)
 
-    def run_rails_command(self, command, timeout=RUNCMD_TIMEOUT):
+    def run_rails_command(self, command, timeout=RUNCMD_TIMEOUT, **kwargs):
         logger.info("Running rails command `%s`", command)
-        return self.run_command('cd /var/www/miq/vmdb; bin/rails runner {}'.format(command),
-            timeout=timeout)
+        return self.run_command('/var/www/miq/vmdb/bin/rails runner {}'.format(command),
+            timeout=timeout, **kwargs)
 
-    def run_rake_command(self, command, timeout=RUNCMD_TIMEOUT):
+    def run_rake_command(self, command, timeout=RUNCMD_TIMEOUT, **kwargs):
         logger.info("Running rake command `%s`", command)
-        return self.run_command('cd /var/www/miq/vmdb; bin/rake {}'.format(command),
-            timeout=timeout)
+        return self.run_command(
+            '/var/www/miq/vmdb/bin/rake -f /var/www/miq/vmdb/Rakefile {}'.format(command),
+            timeout=timeout, **kwargs)
```

Comment:
@mshriver @psav "cd" is removed here too and invoking the rake by its absolute path.